sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/persons/EditPersonPage.java

```diff
@@ -103,8 +103,16 @@
           "//*[contains(text(),'The case person was added as an event participant to the selected event.')]");
   public static final By SEE_EVENTS_FOR_PERSON =
       By.cssSelector("div#See\\ events\\ for\\ this\\ person");
+  public static final By SEE_CASES_FOR_PERSON = By.id("See cases for this person");
+  public static final By SEE_CONTACTS_FOR_PERSON = By.id("See contacts for this person");
+  public static final By EDIT_CASES = By.id("edit-case-0");
```

Comment:
please add also an identifier  _BUTTON _ICON etc, something to understand exactly which element is in UI